src/components/Queries/Queries.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
These aren't components, so they would not belong in the components directory.

Please place them in either `src/graphql`

or `src/queries`